examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,69 @@
+"""
+Extracting brain signal from spheres
+====================================
+
+This example extract brain signals from spheres described by the coordinates
+of their center in MNI space and a given radius in millimeters. In particular,
+this example extracts signals from Default Mode Network regions and compute a
```

Comment:
a bit of repetition here, may be you don't need to specify you are focusing on the DMN